lib/swaggerRefCompiler.js

```diff
@@ -0,0 +1,150 @@
+var fs = require('fs');
+var path = require('path');
+var refCompilerConfig;
+var swaggerDir;
+var thisLogger;
+
+exports.compileSpecs = function (logger, config) {
+    refCompilerConfig = config.get('refCompiler');
```

Comment:
just realized this was not inside the `swagger` config block.

I think it should be.

the default should be for `typeof config.get('swagger').refCompiler === 'object'` to be `false`
which means that setting it to `false` or `"off"` or just omitting it, is fine

when it's configured, however, it should be the configuration object itself